### DISCUSSION TITLE
Is anyone working on mouse support?

### DISCUSSION THREAD

**Discussion by jfed6000** [Hamberthm/esp32-bt2ps2#13] [General]
Just curious if this is being worked on yet.

**Comment by Hamberthm** [comment]
Hi @jfed6000 !

You asked just in time, the answer is YES!! As I write this I'm finishing writing mouse support. Next version will have:

> Mouse support via a separate PS/2 cable (for combined ports, you can use a Y-splitter cable or make your own).
> Multi-device support, so you can connect a mouse+keyboard, etc.
> Multimedia key support (CCONTROL reports).
> Plenty of bug fixes and optimization.

You can expect a release before the end of the week. For building, now ESP-IDF v5.3.0 is needed, otherwise multi-device support won't work.

Will be back soon ;)

**Reply by jfed6000** [reply]
That's awesome!  Thanks so much!

**Comment by Hamberthm** [comment]
@jfed6000 

As promised! Now out: https://github.com/Hamberthm/esp32-bt2ps2/discussions/14

**Comment by jfed6000** [comment]
Was showing off the esp32-bt2ps2 this weekend at Tandy Assembly.  You may have some more users soon.

**Reply by Hamberthm** [reply]
> Was showing off the esp32-bt2ps2 this weekend at Tandy Assembly. You may have some more users soon.

Hey, that's awesome! I did this project just for things like this! Thanks!
